src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String description;
+    protected String type;
+    protected boolean isDone;
+    private static String line = "__________________________________________________________";
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        this.type = "0";
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String getType() {
+        return (type);
+    }
+
+    public String toString() {
+        return ("[" + type + "][" + getStatusIcon() + "] " + description);
+    }
+
+    public void markAsDone(String action) {
```

Comment:
```suggestion
    public void setDone(String action) {
```
since you are changing one of the variables in this class, it is preferred to name similar to getters/setters like you used in `getType`